hera_mc/cm_hookup.py

```diff
@@ -153,7 +153,7 @@ def get_hookup(self, hpn, rev, port, at_date, state_args, exact_match=False):
 
         # Get all the appropriate parts
         parts = self.handling.get_part_dossier(hpn=hpn, rev=rev, at_date=self.at_date, exact_match=exact_match)
-        hookup_dict = {}
+        hookup_dict = {'hookup':{}}
```

Comment:
Does this mean that `hookup_dict` is a nested dict that can contain several hookups? If so, I think the docstring needs to be updated to reflect this -- it looks like the keys of hookup_dict are not keyed on hpn:port anymore.

It looks like there are now two keys in hookup_dict: `hookup` and `columns`. Is that right? The docstring should explain what they are.